apps/site/lib/site_web/controllers/bus_stop_change_controller.ex

```diff
@@ -0,0 +1,56 @@
+defmodule SiteWeb.BusStopChangeController do
+  use SiteWeb, :controller
+
+  alias Alerts.Alert
+
+  def index(conn, _params) do
+    if Laboratory.enabled?(conn, :bus_changes) do
+      conn
+      # Don't let Google crawl this page
+      |> put_resp_header("x-robots-tag", "noindex")
+      # |> assign(:meta_description, "TBA")
+      |> assign(:alerts, bus_stop_alerts(conn.assigns.date_time))
+      |> render("index.html", %{})
+    else
+      redirect(conn, to: alert_path(conn, :index))
+    end
+  end
+
+  """
+  Identify the set of alerts most indicative of bus stop relocations and closures.
+  This is a WIP.
+  """
+
+  defp bus_stop_alerts(date_time) do
+    Alerts.Repo.all(date_time)
+    |> Enum.filter(fn %Alert{
+                        active_period: _active_period,
+                        effect: effect,
+                        lifecycle: _lifecycle
+                      } = alert ->
+      is_closed? = effect in [:stop_closure, :stop_moved, :service_change]
+
+      other_adverse_effects? =
```

Comment:
Will this list need to be updated occasionally? Are these effects stored somewhere else?

Reply:
Yeah, this will probably change in a future iteration. Keeping everything here for now.